org/views.py

```diff
@@ -143,4 +143,41 @@ def Permission_Set_List(request,org_id):
             }
             response_object.append(value)
         return Response(response_object, status.HTTP_200_OK)
-    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
\ No newline at end of file
+    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
+
+
+
+@swagger_auto_schema(
+    operation_id="edit_org",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='PUT',
+    request_body = EditOrgSerializer,
+    responses={
+        '200': set_example(responses.update_org_200),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '403': set_example(responses.admin_access_403),
+        '500': set_example(responses.empty_fields_500)
+    }
+)
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def EditOrg(request,org_id):
+    org_count = Org.objects.filter(pk=org_id).count()
+    if org_count>0:
+        org = Org.objects.get(pk=org_id)
+        user = request.user
+        group = Group.objects.get(name="Admin",org=org)
+        isadmin = True if Member.objects.filter(user=user,group=group).count()>0 else False
+        if isadmin:
+            if request.method == "PUT":
+                serializer = EditOrgSerializer(org,data=request.data)
+                if serializer.is_valid():
+                    serializer.save()
+                    return Response(responses.update_org_200,status.HTTP_200_OK)
+                else:
+                    return Response(responses.empty_fields_500,status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
```suggestion
                    return Response(serializer.errors,status.HTTP_400_BAD_REQUEST)
```
:rofl: :rofl: :rofl:  **We never return http 500 by any means**. Since it is a fault on client side you can use something from the 400 series of error. Refer more here https://www.restapitutorial.com/httpstatuscodes.html/

Reply:
Fixed in ```views.py``` and ```test_edit_org_api.py```.